src/Dont/DontClone.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dont;
+
+use Dont\Exception\NonCloneableObject;
+use Dont\Exception\TypeError;
+
+trait DontClone
```

Comment:
I'm only going by the README.md which you obviously haven't read. It states... that I am completely wrong and should be ignored. Sorry.

I would still prefer to see code such as:

```
namespace Asgrim\Sucks;

class MyEntity
{
    use \Dont\Clone;
}
```